### PR TITLE
fix #78074 stocksregister.com

### LINKED ISSUE #78074: stocksregister.com
### Issue URL (Ads)
 [https://stocksregister.com/2021/03/20/analysts-predict-a-surge-in-axcelis-technologies-inc-nasdaqacls/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fstocksregister.com%2F2021%2F03%2F20%2Fanalysts-predict-a-surge-in-axcelis-technologies-inc-nasdaqacls%2F)
### Comment
Username: @ble0
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://stocksregister.com/
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> https://userscripts.adtidy.org/release/adguard-wot/1.0/wot.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (30), <br>Disable cache for third-party requests, <br>Block WebRTC, <br>Block Push API, <br>Block Location API, <br>Block Flash, <br>Block Java, <br>Hide your Referrer from third-parties (stocksregister.com), <br>Hide your User-Agent (), <br>Hide your IP address (209.141.53.254)
DNS filtering: | server: `sdns://AQcAAAAAAAAAFDE3Ni4xMDMuMTMwLjEzNjo1NDQzILXoRNa4Oj4-EmjraB--pw3jxfpo29aIFB2_LsBmstr6JTIuZG5zY3J5cHQudW5maWx0ZXJlZC5uczEuYWRndWFyZC5jb20`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Chinese, <br/>AdGuard Japanese
Other extensions used: | Nothing

### PR DESCRIPTION
**Pull request by wout-codes**
#78074
Should I add this instead of the current one?
`stocksregister.com##.td-post-content > h3[style] a[class^="manual_trigger"]`

### CODE REVIEW

File: AnnoyancesFilter/sections/self-promo.txt [hunk 1]
```diff
@@ -911,6 +911,7 @@ subbota.ua##.textwidget > div#slidebox
 justnutritive.com##.top-promotion
 forum.doctorhead.ru###content > div[id="secondary_navigation"] ~ noscript + center
 apteka.ru##.apteka-adv
+stocksregister.com##a[class^="manual_trigger"]
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:914]: Something like that `stocksregister.com##.td-post-content > h3[style] a[class^="manual_trigger"]` should be ok.

File: EnglishFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78074
+stocksregister.com#?#div[id^="stock-"]:has(> p[style] > a[href*="/aff_c?offer_id="])
```
Comment (zloyden) [inline EnglishFilter/sections/css_extended.txt:5]: I'd use `stocksregister.com#?#.td-post-content > div[id^="stock-"]:has(> p[style] > span[style]:contains(Sponsored))`.

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: I'd use `stocksregister.com#?#div[id^="stock-"]:has(> p[style] > a[href*="/aff_c?offer_id="])`. It's narrowed enough already.

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/self-promo.txt:914]: What's the problem here: `stocksregister.com##a[class^="manual_trigger"]`. `manual_trigger` appears only once, so I do not think specification is needed. Narrowed enough.

Comment (zloyden) [reply AnnoyancesFilter/sections/self-promo.txt:914]: This rule is more accurate and it shouldn't cause false positives.


> appears only once

Not really safe.

File: EnglishFilter/sections/css_extended.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt:5]: It `a[href*="/aff_c?offer_id="]` would be great if we didn't have that `Sponsored`.

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: C'mon it's the same thing.

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/self-promo.txt:914]: Okay.

File: EnglishFilter/sections/css_extended.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt:5]: Not really. I've already explained the difference.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
By the way, thank you for participating and trying to make the repository better.

**Pull request merged**